optimum/intel/openvino/modeling_base.py

```diff
@@ -195,6 +234,12 @@ def _save_pretrained(self, save_directory: Union[str, Path]):
             save_directory (`str` or `Path`):
                 The directory where to save the model files.
         """
+
+        if self.compile_only:
+            logger.warning(
+                "`compile_only` does not support model saving on disk, you already should have preconverted model"
+            )
+            return
```

Comment:
not a fan of logging a warning and returning nothing when the user explicitly calls `save_pretrained`

Reply:
what will be prefered by you, raise error? There is no entry that can be loaded back after call save_pretrained, that is why I do not provide opportunity to save for avoiding next loading errors. Besides that this mode assumes that user already have saved on disk model before, so it just will create copy of already saved model

Reply:
for all methods or only for saving?

Reply:
Would make sense for all methods to avoid any confusion

Reply:
done